packages/split-view/stories/split-view.stories.ts

```diff
@@ -0,0 +1,257 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { html, TemplateResult } from '@spectrum-web-components/base';
+
+import '../sp-split-view.js';
+
+export default {
+    title: 'Split View',
+    component: 'sp-split-view',
+};
+
+export const Horizontal = (): TemplateResult => {
+    return html`
+        <sp-split-view style="height: 200px" primary-default="100">
+            <div>First panel</div>
+            <div>Second panel</div>
+        </sp-split-view>
+    `;
+};
+
+export const HorizontalResizable = (): TemplateResult => {
+    return html`
+        <sp-split-view
+            resizable
+            primary-min="50"
+            primary-max="Infinity"
```

Comment:
Same suggestion as the docs about the use of `Infinity`.

Reply:
replaced Infinity with real number